ovos_plugin_manager/templates/language.py

```diff
@@ -1,10 +1,14 @@
+from ovos_config.config import Configuration
+
 
 class LanguageDetector:
     def __init__(self, config=None):
-        self.config = config or {}
-        self.default_language = self.config.get("lang") or "en-us"
+        config_core = Configuration()
+        self.config = config or config_core.get('language')
```

Comment:
this was only introduced in neon, should it be documented somewhere since it doesnt exist in other cores yet? maybe docstrs are enough

Reply:
It looks like most of the plugin classes use non-default config sections, so maybe this belongs in the README?